iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/variant.inl

```diff
@@ -63,6 +64,12 @@ inline constexpr variant<Types...>::variant(T&& arg) noexcept
 {
 }
 
+// AXIVION Next Construct AutosarC++19_03-A13.3.1 : False positive. Overloading excluded via std::enable_if in
+// template <typename T,
+// typename = std::enable_if_t<!std::is_same<std::decay_t<T>, variant>::value>,
+// typename std::enable_if_t<!internal::is_in_place_index<std::decay_t<T>>::value, bool> = false,
+// typename std::enable_if_t<!internal::is_in_place_type<std::decay_t<T>>::value, bool> = false>
+// constexpr explicit variant(T&& arg) noexcept;
```

Comment:
Well, in my opinion it's part of the justification. The rule complains that a function that contains forwarding reference as its argument is overloaded. The function meant is (will correct this in the comment)
```
template <typename T>
typename std::enable_if<!std::is_same<T, variant<Types...>&>::value, variant<Types...>>::type&
operator=(T&& rhs) noexcept
```
which is not overloaded by the move and copy assignment operator because of the `std::enable_if<!std::is_same<T, variant<Types...>&>::value, variant<Types...>>`.